### DISCUSSION TITLE
Upgrade 2.5.1 -> 2.6.1, enter key is not working

### DISCUSSION THREAD

**Discussion by JungHyeonSub** [JetBrains/ideavim#737] [General]
Hello,

I upgraded ideavim from 2.5.1 to 2.6.1 in the morning. 
<img width="603" alt="image" src="https://github.com/JetBrains/ideavim/assets/7709807/3865ce42-73a7-4ccb-bc71-464308c79229">
Enter key is not working even if I press Enter key. First of all, I tested pycharm program. I thought settings is broken or something else. In intellij program enter key is working on ideavim version at 2.5.1. After upgrading it, enter key is broken.

ref. 2.6.0 is also not working

What am I supposed to fix this bug?

**Comment by actlikewill** [comment]
same here :cry:  I updated this morning and enter key and backspace are not working. Also new line is not working i.e. pressing O while in normal mode

**Comment by robertpro** [comment]
I can't even double click on the whole word, is there a way to downgrade? this is really slowing me down...

**Comment by dnguyenzd** [comment]
I can't undo changes by typing `u` after upgrading too

if you want to downgrade, go to [plugin page](https://plugins.jetbrains.com/plugin/164-ideavim/versions/stable), download earlier version, and from IDE, choose Plugins -> Settings -> Install Plugin from Disk

**Comment by Catizard** [comment]
can't use undo too.
Another issue was I can't use / or ? in visual mode when I upgrade to 2.5
downgrade the plugin helps me (2.6 -> 2.1)

**Comment by andyd0** [comment]
Yeah, pretty much broken for me even with disabling Ideavim. Kind of weird to release a new update when basic functionality is completely broken

**Comment by AlexPl292** [comment]
The 2.6.0 and 2.6.1 versions of IdeaVim are broken. Please update to the latest 2.6.2 version.
